Source/Atomic/UI/UIWidget.h

```diff
@@ -273,6 +273,46 @@ class UIWidget : public Object, public tb::TBWidgetDelegate
     void Enable();
     void Disable();
 
+    // Font Description
+    void SetFontId(const String& fontId);
+    void SetFontId(tb::uint32 fontId);
```

Comment:
Can we use unsigned here instead of tb::uint32, like to hide as much tb in the engine API as possible.

To select which overload to use for script bindings, it can be added to: 

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/Packages/Atomic/UI.json#L13

for example:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Script/Packages/Atomic/Input.json#L7

If the selected overload is the non-string version, I am not 100% sure how the signature will show up for "unsigned" could be just that or "unsigned int", though if you switch the parameter to an int it will be "int"